docs/platforms/javascript/guides/nuxt/features/pinia.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Pinia
+description: "Learn about enabling Sentry's Pinia plugin in Nuxt."
+---
+
+To capture Pinia state data, add `trackPinia` to your client-side Sentry configuration:
+
+```javascript {3} {filename:sentry.client.config.ts}
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  trackPinia: true // `true` will apply default options
+});
+```
+
+## Normalization Depth
+
+By default, Sentry SDKs normalize any context to a depth of 3. You may want to increase this for sending Pinia states by passing `normalizeDepth` to the `Sentry.init` call:
+
+```javascript {4} {filename:sentry.client.config.ts}
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  trackPinia: true,
+  normalizeDepth: 10, // Or however deep you want your state context to be.
+});
+```
+
+## Options
+
+When enabling `trackPinia` with `true`, all default options are automatically applied. To configure the Pinia plugin manually, pass an options object to `trackPinia`.
+
+<Alert level="warning" title="Note">
+
+While we try our best to filter out Personally Identifiable Information (PII) such as user passwords, we advise against sending sensitive information to Sentry.
+
+</Alert>
+
+
+### `attachPiniaState` (Boolean)
+
+This is used to attach Pinia state to Sentry events. By default, this is set to `true`. If you don't want to attach Pinia state to events being sent to Sentry, set this to `false`.
+
+```javascript {2} {filename:sentry.client.config.ts}
+trackPinia: {
+  attachPiniaState: false
+}
+```
+
+### `addBreadcrumbs` (Boolean)
+
+This is used to add breadcrumbs to Sentry events. By default, this is set to `true`. If you don't want to add breadcrumbs to events being sent to Sentry, set this to `false`.
+
+```javascript {2} {filename:sentry.client.config.ts}
+trackPinia: {
+  addBreadcrumbs: false
+}
+```
+
+### `actionTransformer` (Function)
+
+This can be used to remove sensitive information from Pinia actions. The first parameter passed to the function is the Pinia action name. We send all actions by default, if you don't want an action name sent to Sentry, use `return null`.
```

Comment:
```suggestion
This can be used to remove sensitive information from Pinia actions. The first parameter passed to the function is the Pinia action name. We send all actions by default, if you don't want an action name sent to Sentry, use `return null`:
```